src/main/preferences/schema.json

```diff
@@ -29,6 +29,11 @@
     "type": "boolean",
     "default": false
   },
+  "wordWrap": {
```

Comment:
Sorry for review this PR so late, how about rename it to `wordWrapInToc`?

Reply:
No problem.